Sources/SPTDataLoader.m

```diff
@@ -252,6 +314,14 @@ - (void)requestIsWaitingForConnectivity:(SPTDataLoaderRequest *)request
         [self executeDelegateBlock:^{
             [self.delegate dataLoader:self requestIsWaitingForConnectivity:request];
         }];
+    } else {
+        [self executeDelegateBlock:^{
+            void (^handler)(SPTDataLoaderResult *) = [self.requestHandlers objectForKey:request];
+            handler([SPTDataLoaderResult fromRequest:request kind:SPTDataLoaderResultIsWaitingForConnectivity]);
```

Comment:
Always check for `nil` before invoking a block. If `handler == nil`, this will crash.